dstavis/tasterx
# Feature/router and prescription

What's this PR do?
- Include's Anna's PR to the first branch - quoted below
- Adds prescription component
- Set up router to navigate between pages
- Adds useEffect hook to reset App state when the form component mounts
- Adds some error handling so that the submit button is disabled until a tv show is selected
Where should the reviewer start?
- Start in App and follow flow of components
- Note useEffect hook which also makes use of [useRef](https://betterprogramming.pub/stop-lying-to-react-about-missing-dependencies-10612e9aeeda) hook to prevent an infinite loop
Background info:
- More error handling needs to be implemented

```
What this PR does:

Adds the search form and searching functionality
Assigned data to state in the App, does not add any new display
How should this PR be tested:

Test the form with the dev tools open to see state change in App
The state will update with data fetched from TV maze
Anything else to know:

Utility folder and file added
API call file and folder added
```
